recipe/bld.bat

```diff
@@ -1,15 +1,11 @@
 
 
 set CVXOPT_MSVC=1
+:: the cvxopt builder for suitesparse looks for static ".lib" libraries (not ".dll"!)
 set "CVXOPT_BLAS_LIB_DIR=%LIBRARY_PREFIX%\lib"
-if "%blas_impl%" == "mkl" (
-    set CVXOPT_BLAS_LIB=mkl_rt
-    set CVXOPT_LAPACK_LIB=mkl_rt
-)
-if "%blas_impl%" == "openblas" (
-    set CVXOPT_BLAS_LIB=openblas
-    set CVXOPT_LAPACK_LIB=openblas
-)
+
+set CVXOPT_BLAS_LIB=blas
+set CVXOPT_LAPACK_LIB=lapack
```

Comment:
@isuruf, the switch you removed comes from [anaconda](https://github.com/AnacondaRecipes/cvxopt-feedstock/blob/master/recipe/bld.bat#L3-L10), not from me (even though the recipe was [not building](https://github.com/AnacondaRecipes/cvxopt-feedstock/blob/master/recipe/meta.yaml#L21) for openblas). If I understand [this comment](https://github.com/conda-forge/cvxopt-feedstock/issues/41#issuecomment-604513330) correctly, @msarahan wrote it.

My experience (from 10+ attempts to build this in various configurations), is that `mkl_rt` is necessary, at least for the way that `cvxopt` is set up to build `suitesparse`. If you really want to build suitesparse with netlib only, it might be necessary to patch the `suitesparse`-makefiles to recognise other blas variants correctly, if passing [`BLAS_LIB_DIR`](https://github.com/cvxopt/cvxopt/blob/master/setup.py#L215) and the `.lib`-names is not sufficient.

Suitesparse also [cautions](https://github.com/DrTimothyAldenDavis/SuiteSparse#about-the-blas-and-lapack-libraries) against building with non-mkl for performance reasons:
> ## About the BLAS and LAPACK libraries ##
>
> NOTE: Use of the Intel MKL BLAS is strongly recommended. A recent OpenBLAS can result in severe performance degradation. The reason for this is being investigated, and this may be resolved in the near future. Ignore the comments about OpenBLAS in the various user guides; those are out of date.
>